pyproject_prod.toml

```diff
@@ -0,0 +1,19 @@
+[tool.poetry]
```

Comment:
What's the point of this file?

You should be able to stick to just pyproject.toml. Move dev dependencies like pytest and selenium into the `dev-dependencies` section and then the prod image can install with `poetry install --no-dev` to skip them.